tests/vendor-details-test.js

```diff
@@ -31,27 +30,27 @@ describeApplication('VendorDetails', function() {
     });
 
     it('displays the total number of packages', function() {
-      expect(VendorDetailsPage.numPackages).to.equal(vendor.packagesTotal);
+      expect(VendorDetailsPage.numPackages).to.equal(`Total Packages${vendor.packagesTotal}`);
```

Comment:
we copied this style in the package details pr, and i think it's ok to punt on for now, but we should probably figure out a better way to match these eventually.